### PR TITLE
MW-250: Helpers Documentation

### PR DESCRIPTION
**Pull request by bcanedo4**
Created helpers.html Documentation.

### CODE REVIEW

**Comment by jasmine1569** [comment]
Merge dev into ur branch. Also format your helpers page to look like the other pages.

**Comment by alexhernandez** [comment]
Not sure why this is here but consider removing the following:

```
// Image
.image--center{
  margin: auto;
  display: block;
}

// Box
.box {
  height: 50px;
  width: 50px;
}

.box__no-float {
  height: 50px;
  width: 50px;
}

.background {
  height: 200px;
  width: 35%;
  float: left;
  margin-bottom: 20px;
}

// Helps Documentation display white typography
.highlight {
  background-color: black;
}

.highlight-off {
  background-color: white;
}
```

**Comment by alexhernandez** [comment]
For Colors, group similar colors together:
```
// Background Colors
.bg--teal { background-color: $teal; }
.bg--blue { background-color: $blue; }
.bg--blue-dark { background-color: $blue-dark}
```

**Comment by alexhernandez** [comment]
Have a general `.clearfix` class that provides all the solutions. We don't need to all the ones you provided.

See: https://v4-alpha.getbootstrap.com/utilities/clearfix/

**Comment by alexhernandez** [comment]
If we don't need the margin helpers lets remove it. If we do make sure there is a consistent syntax.

File: dist/css/meta-ui.css [hunk 1]
```diff
@@ -3739,19 +3739,13 @@ ol {
...
-  clear: left !important;
-}
-
-.clear-right {
-  clear: right !important;
-}
-
-.clear-both {
-  clear: both !important;
+.clearfix::after {
```
Comment (jasmine1569) [inline dist/css/meta-ui.css]: if this is a mixin then it should go into _mixin.scss, not the _helpers.scss

**Comment by bcanedo4** [comment]
Oh yeah I asked Alex about that and he wanted me to create the
meta-docs.scss file for css classes that are for documentation, but not for
the UI kit.

**Comment by jasmine1569** [comment]
Ok but you still need to make a mixins folder and put the clearfix in there instead of the helpers.scss

**Pull request merged**